plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -31,6 +31,8 @@ function getContentFromConfluencePage(url, content) {
 
     // Remove any table of contents
     $('.toc').remove();
+    // remove jira issue table
+    $('.jira-table.conf-macro.output-block').remove();
```

Comment:
why not just .jira-table?

Reply:
Could anyone use `jira-table` in some other context where it makes sense to keep it? I wasn't 100% sure.

Reply:
yea, more specific is good since its auto generated anyways.

Reply:
I'm not sure we can easily do it on plugins-api, but since all wiki fetching is done by plugin-site for now, i'm happy with this.